packages/vscode-extension/src/managers/databaseDriverManager.ts

```diff
@@ -0,0 +1,82 @@
+import { initializeDriver } from '@neo4j-cypher/schema-poller';
```

Comment:
the driver manager wraps a driver instance and ensures that we maintain a single driver instance for a given connection, since initializing drivers is quite expensive

the driver will be reinitialized if either the url or username changes.. my approach to this feels a little clunky.. what if a password changes? perhaps I can store a secure hash containing the url, username _and_ password using the crypto package